apps/trackers/tests/test_file_offer.py

```diff
@@ -63,6 +71,7 @@ def test_trackers_file_offer_invalid(
         ps_module_regular = PsModuleFactory(name="regular-module")
         affect = AffectFactory(
             flaw=flaw,
+            impact=impact,
```

Comment:
The impact does not seem to have any effect on the test. Could there be just LOW hard-coded here? Or is there any reason for the parametrizing?

Reply:
There is a validation to permit only LOW impact flaws with DEFER resolution, and thought parameterizing was better than hardcoding for one special case, and better for diversity of test data.

Reply:
It is already hard-coded, just with LOW for one case and MODERATE for the rest. It is the same number of options as if it was all just LOW and one extra condition above. Is MODERATE important for the other cases?

Reply:
I don't think it is. After this last round of changes, I removed impact from the parameters after all.

Reply:
I mean if it is just directly set to LOW there does not have to be any **if else** conditioning

Reply:
Removed in light of not needing validation.